nss-day-cohort-43/nuttyshell-react-plentyofwitches
# Bl create messages

Overview:
The user can now send a public message.

Testing Instructions:
- Log in as different users and try to send a public message.  The messages should render immediately
-Try and send an empty message.  An empty message should not be sent
